eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -179,6 +179,13 @@ steps:
     configuration: "$(BuildConfiguration)"
     msbuildArguments: "/restore /binarylogger:$(Build.StagingDirectory)\\binlog\\09.SignAssemblies.binlog"
 
+- task: MSBuild@1
+  displayName: "Copy signed assemblies to localize artifacts folder"
+  inputs:
+    solution: "build\\loc.proj"
+    configuration: "$(BuildConfiguration)"
+    msbuildArguments: "/restore /t:CopyBinariesToLocalizationArtifacts /binarylogger:$(Build.StagingDirectory)\\binlog\\09.LocalizeFolderAssemblies.binlog"
```

Comment:
nitpick:  there's already a number `09` binlog, and when sorted, this binlog will be listed before the other `09`, even though it runs afterwards.

I know there are quite a few lines to modify to give each a unique number, but we don't add or remove build steps very often, so I don't think it's unreasonable to update them all.

Reply:
Yes, I used 09 because it's directly correlated with the signing. I didn't want to invent a new thing like 09a and 09b, since I figured that'd be controversial and also not very important.
I agree, changing all these numbers every time we make an edit is probably more confusing to us in the long run.

Reply:
I proposed the opposite. I think we should edit every number every time we make an edit.

It's like the write once vs read many argument for product code. We should optimize for investigating build failures, rather than making it easier to edit occasionally.